DemoApp/DemoApp/Views/TMButtonPage.xaml.cs

```diff
@@ -47,4 +48,17 @@ private void ImagePositionChanged(object sender, CheckedChangedEventArgs e)
             _buttonPageViewModel.SelectedImageOption = radioButton.Value.ToString();
         }
     }
+
+    private void isFullWidth_Toggled(object sender, ToggledEventArgs e)
+    {
+        if (e.Value)
+        {
+            _buttonPageViewModel.IsFullWidthAlignment = LayoutOptions.FillAndExpand;
```

Comment:
IsFullWidthAlignment is not a boolean property so please rename the valid property name

Reply:
Renamed